src/test/java/wirelessmesh/IntegrationTest.java

```diff
@@ -0,0 +1,168 @@
+package wirelessmesh;
+
+import com.akkaserverless.javasdk.AkkaServerless;
+import com.akkaserverless.javasdk.testkit.junit.AkkaServerlessTestkitResource;
+import com.akkaserverless.javasdk.testkit.junit.jupiter.AkkaServerlessTest;
+import io.grpc.StatusRuntimeException;
+import org.junit.ClassRule;
+import org.junit.Test;
+import wirelessmeshdomain.Wirelessmeshdomain.*;
+import wirelessmeshservice.WirelessMeshServiceClient;
+import wirelessmeshservice.Wirelessmeshservice.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@AkkaServerlessTest
```

Comment:
This annotation is for JUnit 5 (to register the extension). Not needed for JUnit 4.

Reply:
Ah, I didn't realize that. I wonder how we can make that discoverable. Perhaps we should just update the example project to JUnit 5.

Reply:
(done in #2, not sure if it makes sense)

Reply:
Yeah, not sure how to make it more clear. It's under a `junit.jupiter` package, and there are separate javadocs for `@AkkaServerlessTest`. Doesn't hurt to add the `@AkkaServerlessTest` annotation, just doesn't do anything.